frame/support/src/dispatch.rs

```diff
@@ -2097,6 +2097,7 @@ macro_rules! decl_module {
 					$(
 						$call_type::$fn_name( $( ref $param_name ),* ) => {
 							let __pallet_base_weight = $weight;
+							let __pallet_pov_size = 0;
```

Comment:
yes, let's not support stuff here finally :D

Reply:
Lol yes, I got too lazy to support any more new syntax there, so I stubbed this out with just 0.